### DISCUSSION TITLE
Scrypted Proxy for Home Assistant

### DISCUSSION THREAD

**Discussion by hazza64** [koush/scrypted#889] [Ideas]
Previously, when I was using Frigate, it had a proxy that meant you could have the CPU-intensive stuff like NVR + motion detection, etc on one computer/server, and the interface running within Home Assistant.
I noticed that while there is a Scrypted interface for Home Assistant, you need to have installed the entire Scrypted stack within HA to access the functions like Tiles, etc.
Is there a plan or existing method for running the frontend/proxy only as an addon for HA?

**Comment by koush** [comment]
There's no plans to do a HA proxy for Scrypted. HA probably has support for generalized proxies. You can find and use one of those.

Scrypted NVR will be adding support for HA cards on separate installations.
